Source/Parallelization/WarpXRegrid.cpp

```diff
@@ -156,67 +206,19 @@ WarpX::RemakeLevel (int lev, Real /*time*/, const BoxArray& ba, const Distributi
         ScaleEdges();
         ScaleAreas();
         ComputeDistanceToEB();
+
+        // Since we have reset m_borrowing we need to recompute the extensions.
+        // To do so we need to recompute m_flag_ext_face since it has been changed
+        // since the last time ComputeFaceExtensions has been called. For this reason
+        // we call MarkCells before ComputeFaceExtensions
+        if(WarpX::maxwell_solver_id == MaxwellSolverAlgo::ECT){
+            MarkCells();
+            ComputeFaceExtensions();
+        }
```

Comment:
Sorry I had a closer look and there is something more we need to add, but that should be it. 

```suggestion
        // Since we have reset m_borrowing we need to recompute the extensions.
        // To do so we need to recompute m_flag_ext_face since it has been changed
        // since the last time ComputeFaceExtensions has been called. For this reason
        // we call MarkCells before ComputeFaceExtensions
        if(WarpX::maxwell_solver_id == MaxwellSolverAlgo::ECT){
            // Since we have reset m_borrowing we need to recompute the extensions.
            // To do so we need to recompute m_area_mod and m_flag_ext_face since they have been changed
            // since the last time ComputeFaceExtensions has been called. For this reason
            // we call ComputeFaceAreas, ScaleAreas and MarkCells before ComputeFaceExtensions
            ComputeFaceAreas;
            ScaleAreas;
            MarkCells();
            ComputeFaceExtensions();
        }
```

Reply:
No problem at all. The functions `ComputeFaceAreas()` and `ScaleAreas()` are called in the lines above this block. Should we remove them from there since they are not needed if `WarpX::maxwell_solver_id != MaxwellSolverAlgo::ECT`?